src/python-common/ceph/tests/test_drive_group.py

```diff
@@ -70,8 +70,8 @@ def test_ceph_volume_command_2():
                           db_devices=DeviceSelection(size='200GB:350GB', rotational=False),
                           wal_devices=DeviceSelection(size='10G')
                           )
-    inventory = _mk_inventory(_mk_device(rotational=True)*2 +
-                              _mk_device(rotational=False)*2 +
+    inventory = _mk_inventory(_mk_device(rotational=True, size="300.00 GB")*2 +
+                              _mk_device(rotational=False, size="300.00 GB")*2 +
```

Comment:
... this actually fixes a bug here. by default, drives are `390GB` and thus bigger than `size='200GB:350GB'`. The size attribute never matched.